templates/list_pages/oe-list-pages-selected-facet.html.twig

```diff
@@ -1,9 +1,13 @@
-{% for item in items %}
-  {{ pattern('badge', {
-    'background': 'light',
-    'label': item.label,
-    'url': item.url,
-    'dismissible': true,
-    'attributes': create_attribute({'class': ['me-2', 'fs-6']}),
-  }) }}
-{% endfor %}
+{% if items -%}
+<div class="col-md-12 order-md-2 mt-3">
```

Comment:
Why are we doing this?
For me this has the effect that now each badge is on a new line. I suspect this is not intended.

Reply:
Very dubious indeed.